include/rl_string_view.h

```diff
@@ -1,9 +1,17 @@
 #pragma once
 
-// Rely on the VW implementation of string_view
-#include "vw/common/string_view.h"
+#include "future_compat.h"
 
+#ifdef HAS_STD17
+#  include <string_view>
 namespace reinforcement_learning
 {
-using string_view = VW::string_view;
+using std::string_view;
```

Comment:
I believe string view lite delegates internally to the STL implementation if it exists so this isn't really necessary unless we push this all the way through and not even include/search/install string-view-lite if c++17 is being used.

Reply:
You're right. Looks like VW always uses string-view-lite regardless of C++ version, so I'll make it do that here to maintain consistency.